rocAL/include/augmentations/geometry_augmentations/node_crop_resize.h

```diff
@@ -31,8 +31,8 @@ class CropResizeNode : public CropNode {
     CropResizeNode() = delete;
     void init(float area, float aspect_ratio, float x_center_drift, float y_center_drift);
     void init(FloatParam *area, FloatParam *aspect_ratio, FloatParam *x_drift_factor, FloatParam *y_drift_factor);
-    unsigned int get_dst_width() { return _outputs[0]->info().max_shape()[0]; }
-    unsigned int get_dst_height() { return _outputs[0]->info().max_shape()[1]; }
+    unsigned int get_dst_width() { return _outputs[0]->info().width_height()[0]; }
```

Comment:
Rajy did not want any of the names in tensor class to have width and height, That was the reason max shape was introduced